scripts/pi-hole/js/settings-api.js

```diff
@@ -88,11 +88,13 @@ $(function () {
       $("td:eq(9)", row).html(button);
       if (data.current_session) {
         ownSessionID = data.id;
-        $("td:eq(7)", row).html(
-          '<strong title="This is the session you are currently using for the web interface">' +
-            data.remote_addr +
-            "</strong>"
-        );
+        $("td", row).each(function () {
+          $(this).addClass("text-bold");
+          $(this).attr(
+            "title",
+            "This is the session you are currently using for the web interface"
+          );
+        });
```

Comment:
You don't need the `.each()` loop. You can add the class and title to the row (`tr`). 

```suggestion
        $(row).addClass("text-bold");
        $(row).attr("title", "This is the session you are currently using for the web interface");
```